profiles/angular/skeleton/grails-app/assets/javascripts/@grails.codegen.defaultPackage.path@/app.js

```diff
@@ -0,0 +1,14 @@
+//= wrapped
```

Comment:
I usually put the require for angular in core module with all the other dependencies but I don't have a strong opinion on that either way. Let me know if you want me to move that to `app.js`